docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-107.adoc

```diff
@@ -39,6 +39,8 @@ If your OpenAI API key is exposed, immediately regenerate a new key through your
 3. Find the API keys management page.
 4. If you believe your API key has been compromised or if you wish to invalidate an existing key for any reason, click on the option to regenerate a new API key.
 5. Immediately replace the compromised API key in your application or service with the new API key provided.
+6. Review API access and usage after replacing the API key to identify any unauthorized access or suspicious activities.
```

Comment:
Steps 6 and 7 are not part of generating an API Key - they are 'next steps' or something like Best Practice